### ISSUE TITLE
No module named 'numba.decorators' after upgrade to numba 0.50

### ISSUE THREAD

**Issue by korseby** [librosa/librosa#1172]
Hi all,

I'm on macOS 10.15 and using Python 3.8. Recently, I upgraded my python modules using pip. When using my py3tag-script (which can be found here: https://github.com/korseby/py3tag), I noticed that librosa seems to import `decorators` from `numba` which seem to have either removed or renamed from `numba` with the latest version. Downgrading numba to `0.48` (from `0.50`) solved the problem for now.

Here's the error message when having numba 0.50 installed:
```Python
Traceback (most recent call last):
  File "/opt/bin/py3tag.py", line 16, in <module>
    import librosa
  File "/opt/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/librosa/__init__.py", line 12, in <module>
    from . import core
  File "/opt/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/librosa/core/__init__.py", line 125, in <module>
    from .time_frequency import *  # pylint: disable=wildcard-import
  File "/opt/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/librosa/core/time_frequency.py", line 11, in <module>
    from ..util.exceptions import ParameterError
  File "/opt/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/librosa/util/__init__.py", line 77, in <module>
    from .utils import *  # pylint: disable=wildcard-import
  File "/opt/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/librosa/util/utils.py", line 15, in <module>
    from .decorators import deprecated
  File "/opt/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/librosa/util/decorators.py", line 9, in <module>
    from numba.decorators import jit as optional_jit
ModuleNotFoundError: No module named 'numba.decorators'
```

**Software versions**
```
macOS-10.15.5-x86_64-i386-64bit
Python 3.8.3 (default, May 31 2020, 12:24:29)
[Clang 11.0.0 (clang-1100.0.33.17)]
NumPy 1.19.0
SciPy 1.4.1
librosa 0.7.2
INSTALLED VERSIONS
------------------
python: 3.8.3 (default, May 31 2020, 12:24:29)
[Clang 11.0.0 (clang-1100.0.33.17)]

librosa: 0.7.2

audioread: 2.1.8
numpy: 1.19.0
scipy: 1.4.1
sklearn: 0.23.1
joblib: 0.15.1
decorator: 4.4.2
six: 1.15.0
soundfile: 0.10.3
resampy: 0.2.2
**numba: 0.50.0**

numpydoc: None
sphinx: None
sphinx_rtd_theme: None
sphinxcontrib.versioning: None
sphinx-gallery: None
pytest: None
pytest-mpl: None
pytest-cov: None
matplotlib: 3.2.2
presets: None
```

**Comment by lostanlen** [comment]
Hello. We are aware of the problem. This issue is a duplicate of #1105 and was closed in #1107.

For more details, please refer to https://github.com/numba/numba/issues/5865

The quick fix is to downgrade to `numba==0.48`

**Comment by Bharadwaj75-ceo** [comment]
> 

Doesn't work it says resampy not compatible.
resampy 0.4.2 requires numba>=0.53 but you have numba 0.48.0
